pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -307,7 +307,7 @@ class Build {
     */
     VersionInfo parseVersionOutput(String consoleOut) {
         context.println(consoleOut)
-        Matcher matcher = (consoleOut =~ /(?ms)^.*OpenJDK Runtime Environment[^\n]*\(build (?<version>[^)]*)\).*$/)
+        Matcher matcher = (consoleOut =~ /(?ms)^.*Runtime Environment[^\n]*\(build (?<version>[^)]*)\).*$/)
```

Comment:
Can we expand the comment at the top to include the example you're widening this to?

Reply:
Updated.